source/intercept-library/executable/CMakeLists.txt

```diff
@@ -7,7 +7,9 @@ add_subdirectory(man)
 
 # Create a static executable from the sub projects.
 set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
-set(CMAKE_EXE_LINKER_FLAGS "-static")
```

Comment:
This is important to have this executable linked statically. (So, we fixed a build time problem, but will have a runtime one.)
If static linking is not feasible on this platform, then it shall not build the targets from `source/intercept-library`.